jenkins/internal-ci/generic/branch/motr.groovy

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env groovy
+// Please configure branch and os_version as string parameters in Jenkins configuration manually.
+pipeline {
+    agent {
+        node {
+            label "docker-${os_version}-node"
+        }
+    }
+
+    triggers {
+        pollSCM '*/5 * * * *'
+    }
+    
+    parameters {
+        choice(
+            choices: ['libfabric' , 'lustre'],
+            description: '',
+            name: 'TRANSPORT')
+    }
+        
+    environment {
+        version = "2.0.0"    
+        env = "dev"
+        component = "motr"
+        release_dir = "/mnt/bigstorage/releases/cortx"
+        build_upload_dir = "$release_dir/components/github/$branch/$os_version/$env/$component"
+
+        // Dependent component job build
+        build_upload_dir_s3_dev = "$release_dir/components/github/$branch/$os_version/$env/s3server"
```

Comment:
do we need s3?

Reply:
Kept it for support in case we need to use RGW build details here. Need to confirm this with Motr team.

Reply:
Should we keep it then? or you want to confirm it with Motr before megre?

Reply:
Taking confirmation with Motr might take time and I don't think this is blocking change. So I feel even if we merge with this nothing will break.